src/components/Voting/Quadratic.vue

```diff
@@ -54,44 +54,43 @@ watch(selectedChoices.value, currentValue => {
   <div class="mb-3">
     <div v-for="(choice, i) in proposal.choices" :key="i">
       <UiButton
-        class="block w-full mb-2"
+        class="mb-2 flex justify-between items-center w-full overflow-hidden"
         :class="selectedChoices[i + 1] > 0 && 'button--active'"
       >
         <div
-          class="inline-block w-7/12 sm:w-8/12 float-left text-left pr-3"
+          class="text-left pr-3 truncated"
           v-tippy="{
-            content: choice
+            content: choice.length > 20 && isSmallScreen ? choice : null
```

Comment:
Hmmm this will have some edgecases where the text is truncated but no tooltip is shown. But it's still far better than now.

To avoid this completely, we'll have to change the layout entirely at some point, so that we can simply always display the full text without any truncating and tooltips and so on. Especially on mobile tooltips are not so cool.